client/src/components/UI/Modal/Modal.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Modal.css';
+
+const modal = (props) => {
```

Comment:
@simonnguyen3054 You should capitalize the stateless component name 'modal' so that it is 'Modal', that way we are consistent throughout our code.